File: backEstacionamento/Models/NotaFiscal.cs
using System.ComponentModel.DataAnnotations;

public class NotaFiscal
{
    [Key]
    
    public int? _NumeroNota { get; set; }
    public double? _ValorDaNota { get; set; }
    public String? _Cpf{  get; set; }
    public int? _idServico {  get; set; }
    

    //-------------------------------------------------------------------------
    public Cliente? Cliente { get; set; }
    public Servico? Servico{get;set;}
    //-------------------------------------------------------------------------

    public List<Servico>? Servicos {get;set;}
    

    //-------------------------------------------------------------------------

    
}

File: frontEstacionamento/src/app/models/NotaFiscal.ts
import { Cliente } from "./Cliente";
import { Servico } from "./Servico";

export class NotaFiscal
{
    _NumeroNota?: string = "";
    _ValorDaNota: number = 0.0;
    _Cpf: Cliente |undefined;
    _idServico?: Servico | undefined;
}